crates/floresta-wire/src/p2p_wire/node.rs

```diff
@@ -183,21 +183,21 @@ pub struct NodeCommon<Chain: BlockchainInterface + UpdatableChainstate> {
     pub(crate) network: Network,
 }
 
-pub struct UtreexoNode<Context, Chain: BlockchainInterface + UpdatableChainstate>(
-    pub(crate) NodeCommon<Chain>,
-    pub(crate) Context,
-);
+pub struct UtreexoNode<Chain: BlockchainInterface + UpdatableChainstate, NodeModule> {
```

Comment:
35837dcbf97c0f0e2b7e36e910fdcb8f1e814d66

The generic name here wasn't updated